admin/workspace-providers.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Workspace Providers
+description: Learn how workspace providers can minimize latency for users.
+state: beta
+---
+
+Coder allows you to add to your deployment multiple Kubernetes clusters. These
+additional clusters are called **providers**. This can be helpful for
+distributed teams, since users can choose the provider closest to them to
+minimize latency.
+
+## Enabling Workspace Providers
+
+To enable the Workspace Providers UI elements:
+
+1. In the Coder UI, click on your avatar.
+2. In the menu that appears, click **Feature Preview**.
+3. Select **Resource Pools UI** and click **Enable**.
+
+Once enabled, this allows end-users to choose the workspace provider and
+namespace where they want their environment deployed.
+
+## Admin UI
+
+Site managers and admins can view the workspace providers configuruation
```

Comment:
This is up in the air -- as it is, it's not because we don't capitalize image, environments, etc. I lean toward this direction but could def be swayed otherwise